package.json

```
@@ -5,6 +5,7 @@
     "postinstall": "husky install",
     "bootstrap": "yarn run _init:agora-configure && yarn install --frozen-lockfile --force",
     "lint": "yarn workspaces run lint",
+    "lint:global": "lint-staged",
```

```suggestion
    "lint": "lint-staged && yarn workspaces run lint",
```
This would be better